platform/admin/modules/admin-docs-ui/src/main/resources/404.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Missing Docs</title>
+        <style>
+            h1 {
+                margin: 0;
+                font-weight: bold;
+                color: #18bc9c;
+                padding: 10px;
+                font-family: sans-serif;
+            }
+            p {
+                margin: 0;
+                padding: 10px;
+                color: #2c3e50;
+                font-size: 1.3em;
+                font-family: sans-serif;
+            }
+        </style>
+    </head>
+<body>
+    <h1>Documentation Not Found</h1>
```

Comment:
Integrators with their own documentation would be pointing to the incorrect project if we add the link. The released version should also always contain docs so I'm not too concerned about this. It's more of a fallback just in case.